Homework 5.cpp

```diff
@@ -0,0 +1,153 @@
+//
+//  Homework 5.cpp
+//  StudyFall16
+//
+//  Created by Farida Eid on 7/8/17.
+//  Copyright © 2017 Farida Eid. All rights reserved.
+//
+
+#include <iostream>
+#include <vector>
+#include <map>
+#include <unordered_map>
+#include <stack>
+#include <set>
+
+using namespace std;
+
+class Graph
+{
+private:
+    map<char,set<char>> adjList;
```

Comment:
It's more clear like this, right?
I do have to note that your previous version ran faster, but code readability is usually more important.